File: C++/jackolanternjuxtaposition.cpp
#include <iostream>
using namespace std;

int main() {
  // N: num of eye designs 
  // T: num of nose designs
  // M: num of mouth designs
  int n, t, m;
  cin >> n >> t >> m;

  // Output: num of different possible designs
  cout << n * t * m << endl;
}

File: Java/jackolanternjuxtaposition.java
import java.io.*;
import java.util.StringTokenizer;

public class jackolanternjuxtaposition {
  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    StringTokenizer st = new StringTokenizer(br.readLine());

    int n = Integer.parseInt(st.nextToken()); // num of eye designs
    int t = Integer.parseInt(st.nextToken()); // num of nose designs
    int m = Integer.parseInt(st.nextToken()); // num of mouth designs

    System.out.println(n * t * m);  // num of different possible designs
  }
}
